modules/virt-restoring-vm-from-snapshot-cli.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-managing-offline-vm-snapshots.adoc
+
+[id="virt-restoring-vm-from-snapshot-cli_{context}"]
+= Restoring a virtual machine from a snapshot in the CLI
+
+You can restore an existing virtual machine (VM) to a previous configuration by using a VM snapshot .
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Power down the VM you want to restore to a previous state.
+
+.Procedure
+
+. Create a YAML file to define a `VirtualMachineRestore` object that specifies the name of the VM you want to restore and the name of the snapshot to be used as the source.
++
+For example:
++
+[source,yaml]
+
+----
+apiVersion: snapshot.kubevirt.io/v1alpha1
+kind: VirtualMachineRestore
+metadata:
+  name: my-vmrestore <1>
+spec:
+  target:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: my-vm <2>
+  virtualMachineSnapshotName: my-vmsnapshot <3>
+----
+<1> The name of the new `VirtualMachineRestore` object.
+<2> The name of the target VM you want to restore.
+<3> The name of the `VirtualMachineSnapshot` object to be used as the source.
+
+. Create the `VirtualMachineRestore` resource. The snapshot controller updates the status fields of the `VirtualMachineRestore` object and replaces the existing VM configuration with the snapshot content.
++
+[source,terminal]
+----
+$ oc create -f <my-vmrestore>.yaml
+----
+
+.Verification steps
+
+. Verify that the VM is restored to the previous state represented by the snapshot. The `complete` flag must be set to `true`.
++
+[source,terminal]
+----
+$ oc get vmrestore <my-vmrestore>
+----
+
+.Example output
```

Comment:
This example output isn't indented properly with the above step. You'll need to add a "+" on the line above.

Reply:
Done